Update README.md (#105)
Akuli/mantaray: README.md

    $ source env/bin/activate
    $ pip install -r requirements.txt
    $ python3 -m mantaray
    

> For Windows users the procedure type these commands in Command Prompt:

Reviewer: slightly shorter, still contains the same info:
```suggestion
On Windows, run these commands in Command Prompt:
```
